recipes/drupal8-multisite/README.md

```diff
@@ -0,0 +1,152 @@
+# Introduction
+DDEV supports Drupal 8 multisite, however it is not always obvious what changes need to be done to make it work. Additionally, sometimes setup helpful for single-site configuration blocks the desired behavior for a multi-site setup.
+
+This example will show how to setup a basic two-site multisite configuration, covering ddev, database, drush, and other issues. It has been tested on a Mac.
+
+Neither of the sites will be default to ensure that if something is misconfigured, commands will fail against incomplete Drupal site instance instead of silently running against wrong one.
+
+# Initial setup
+
+Let's mostly follow the suggested configuration in [the official DDEV documentation for Drupal 8 setup](https://ddev.readthedocs.io/en/stable/users/cli-usage/#drupal-8-quickstart).
+1. mkdir d8m
+1. cd d8m
+1. ddev config --project-type php
+1. ddev composer create drupal-composer/drupal-project:8.x-dev --stability dev --no-interaction --prefer-dist
+1. ddev config --project-type drupal8
```

Comment:
This guide looks amazing. I'm wondering if the `php` project type would be a better choice, though. The `drupal8` project type tries to re-add `settings.ddev.php` to `sites/default/settings.php` every time. This might be solved by not _having_ a `default` site, though, thus a non-issue? `drupal8` _is_ probably a better project type because I have found that when using `ddev config` to upgrade my configuration, ddev re-detects the project as `drupal8` anyway.

I am planning to set up a multisite config soon, so I will keep this recipe in mind and see if I can contribute anything.